data/json/items/tool/entry_tools.json

```diff
@@ -127,7 +127,12 @@
     "looks_like": "crowbar",
     "description": "A 5 foot long construction prybar.  Essentially a long solid steel rod with a hardened chisel tip.",
     "weight": "8 kg",
-    "to_hit": 1,
+    "to_hit": {
+    "grip": "solid",      // one of "bad", "none", "solid", or "weapon"
```

Comment:
I don't think this part after // is supposed to be here?

Reply:
I removed that. That's what had been causing my syntax error before. When I look at the file now it's not showing up for me.